CMakeLists.txt

```diff
@@ -325,8 +325,8 @@ foreach (CHPLENV_LINE IN LISTS CHPLENV_OUTPUT)
   set(${CHPL_ENV_NAME} ${CHPL_ENV_VALUE} CACHE STRING "overwritten description" FORCE)
 endforeach()
 # We set the cxx standard in the sub folders, strip the standard from the bunch of flags we got
-string(REPLACE "-std=c++14" "" CHPL_HOST_BUNDLED_COMPILE_ARGS "${CHPL_HOST_BUNDLED_COMPILE_ARGS}")
-string(REPLACE "-std=c++14" "" CHPL_HOST_SYSTEM_COMPILE_ARGS "${CHPL_HOST_SYSTEM_COMPILE_ARGS}")
+string(REPLACE "-std=c++17" "" CHPL_HOST_BUNDLED_COMPILE_ARGS "${CHPL_HOST_BUNDLED_COMPILE_ARGS}")
+string(REPLACE "-std=c++17" "" CHPL_HOST_SYSTEM_COMPILE_ARGS "${CHPL_HOST_SYSTEM_COMPILE_ARGS}")
```

Comment:
I'd recommend leaving the `-std=c++17` changes out of this PR (and including them in a later PR once we know we are doing LLVM 16 this release).